protocol/0028-GOVE-governance.md

```diff
@@ -529,7 +529,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 - Proposals are either permitted or rejected according to the following source/destination combinations. (<a name="0028-GOVE-075" href="#0028-GOVE-075">0028-GOVE-075</a>)
 - Transfer amount and fraction of balance are mutually exclusive and will cause the proposal to reject (<a name="0028-GOVE-076" href="#0028-GOVE-076">0028-GOVE-076</a>)
-- Invalid source and destination account types will cause the proposal to reject (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
+- Invalid source and destination account types will not be included in the transaction and will result in a wallet rejection (<a name="0028-GOVE-077" href="#0028-GOVE-077">0028-GOVE-077</a>)
```

Comment:
Wallet rejection is not precise enough; people may choose to not use the vega wallet to submit transactions. It must be rejected by the chain.

Reply:
updated - please check @davidsiska-vega